src/components/TextInput/index.js

```diff
@@ -0,0 +1,58 @@
+import { Component } from 'pet-dex-utilities';
+import './index.scss';
+
+const events = [
+  'inputError',
+  'inputPlaceholderChanged',
+  'inputAssetChanged',
+  'inputDisabled',
+  'inputEnabled',
+];
+
+const html = `
+    <div class="input-text-container">
+      <input class="input-text-container__input" type="text" data-select="input-text" placeholder="">
+  </div>
+`;
+
+export default function InputText() {
```

Comment:
```suggestion
export default function InputText({
placeholder,
suffix,
prefix,
variation, // outlined, standard
}) {
```

os outros não tenho ctz se fazem sentido estarem aqui, mas pode ser que façam! Qualquer coisa chama nois